src/include/miopen/any_solver.hpp

```diff
@@ -243,51 +246,59 @@ struct AnySolver
         float GetWti(const ConvolutionContext& ctx) const override { return value.GetWti(ctx); }
 
         ConvSolution FindSolution(const ConvolutionContext& ctx,
-                                  Db& db,
-                                  const miopen::AnyInvokeParams& invoke_ctx) const override
+                                  PerformanceDb& db,
+                                  const miopen::AnyInvokeParams& invoke_ctx,
+                                  const std::optional<std::string>& perf_cfg) const override
         {
-            return miopen::solver::FindSolution(value, ctx, db, invoke_ctx);
+            return miopen::solver::FindSolution(value, ctx, db, invoke_ctx, perf_cfg);
         };
 
-        std::string GetPerfCfgParams(const ConvolutionContext& ctx, Db& db, std::true_type) const
+        std::optional<std::string>
+        GetPerfCfgParams(const ConvolutionContext& ctx, PerformanceDb& db, std::true_type) const
         {
             using PerformanceConfig = decltype(value.GetDefaultPerformanceConfig(ctx));
             PerformanceConfig config{};
             if(db.Load(ctx.problem, value.SolverDbId(), config))
             {
-                MIOPEN_LOG_I2("Perf Db: Record Loaded: " << value.SolverDbId());
+                MIOPEN_LOG_I2("Perf PerformanceDb: Record Loaded: " << value.SolverDbId());
```

Comment:
"Perf PerformanceDb" -- what dos it mean?

Reply:
Perhaps "Perf PerformanceDb" could be shortened to "PerformanceDb". Which seems equivalent to the previous string here, though I assume the change is to reflect the name of the new C class.